apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -1,11 +1,49 @@
-/*global BridgeServiceMixin */
+/*global bridge,
+         BridgeServiceMixin,
+         BroadcastChannel,
+         Drafts,
+         streamClient,
+         Thread
+*/
 'use strict';
 
-if (!('BridgeServiceMixin' in self)) {
-  importScripts('/services/js/bridge_service_mixin.js');
-}
+/**
+ * Type description for the ConversationSummary object that is return from the
+ * Conversation service.
+ * @typedef {Object} ConversationSummary
+ * @property {number} id Unique identifier of the the conversation.
+ * @property {string} body Short content of the last message in the
+ * conversation.
+ * @property {Array.<string>} participants List of the conversation
+ * participants.
+ * @property {number} timestamp Conversation timestamp.
+ * @property {number} unreadCount Number of conversation messages that user has
+ * not read yet.
+ * @property {string} lastMessageSubject Subject of the last message in the
+ * conversation if it has one.
+ * @property {string} lastMessageType Type of the last message in the
+ * conversation. Can be either 'sms' or 'mms'.
+ * @property {Draft} draft If conversation is draft or has draft, then this
```

Comment:
could be Array<ContactInformaion>; ContactInformation could either be a real contact with contactName/carrierType/etc (but in Inbox we need contactName only, the other fields will be missing) or an unknown address (number/email)
(just a proposal)